x/dispute/keeper/tally.go

```diff
@@ -101,16 +99,20 @@ func (k Keeper) TallyVote(ctx context.Context, id uint64) error {
 	if err != nil {
 		return err
 	}
-	teamDidVote, err := k.Voter.Has(ctx, collections.Join(id, teamAddr.Bytes()))
+	teamVote, err := k.Voter.Get(ctx, collections.Join(id, teamAddr.Bytes()))
+	var teamDidVote bool
```

Comment:
seems redundant, teamVote defaults to false, no? if collections.ErrNotFound then its false otherwise its true.

Reply:
actually nvm, not .Get